src/language/mongoDBService.ts

```diff
@@ -43,6 +50,31 @@ export default class MongoDBService {
     return this._connectionOptions;
   }
 
+  private isSslConnection(connectionOptions: any): boolean {
+    return !!(
+      connectionOptions &&
+      (connectionOptions.sslCA ||
+        connectionOptions.sslCert ||
+        connectionOptions.sslPass)
+    );
+  }
+
+  private loadSslOptionsFromFile(connectionOptions: any): SslFileOptions {
+    const sslOptions: SslFileOptions = {};
+
+    if (connectionOptions.sslCA) {
+      sslOptions.sslCA = fs.readFileSync(connectionOptions.sslCA[0]);
```

Comment:
Should we have this wrapped in a try catch in case it fails? Might be a place where the user has moved the fail so it would be nice to give them a message saying file reading failed.